src/opencv.mk

```diff
@@ -48,7 +48,7 @@ define $(PKG)_BUILD
       '$(1)'
 
     # install
-    $(MAKE) -C '$(1).build' -j '$(JOBS)' install VERBOSE=1
+    $(MAKE) -C '$(1).build' -j '$(JOBS)' install #VERBOSE=1
```

Comment:
opencv is using cmake. if you need proper verbose that will be CMAKE_VERBOSE_MAKEFILE=TRUE. 

maybe could be adding in mxe toolchain. This is only for if one choose to actually have verbose.